releasenotes/notes/openai-fix-logging-extra-endpoints-d6563f7dd1a5828b.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    openai: Resolves some inconsistencies in logs generated by the image and audio endpoints,
+    including filenames, prompts, and not logging raw binary image data. 
```

Comment:
Can we make this a bit more explicit. What inconsistencies were addressed?

Reply:
The inconsistencies are the `filenames, prompts, and not logging raw binary image data` 😄